pkg/apis/elasticserving/v1/paddleservice_status.go

```diff
@@ -55,30 +57,25 @@ func (ss *PaddleServiceStatus) GetCondition(t apis.ConditionType) *apis.Conditio
 }
 
 func (ss *PaddleServiceStatus) PropagateStatus(serviceStatus *knservingv1.ServiceStatus) {
-	conditionType := DefaultPaddleServiceReady
+	if serviceStatus == nil {
+		return
+	}
+	// conditionType := DefaultEndpointReady
 	statusSpec := StatusConfigurationSpec{}
 	if ss.Default == nil {
 		ss.Default = &statusSpec
 	}
-	if serviceStatus == nil {
-		return
-	}
 	statusSpec.Name = serviceStatus.LatestCreatedRevisionName
-	serviceCondition := serviceStatus.GetCondition(knservingv1.ServiceConditionReady)
+	// serviceCondition := serviceStatus.GetCondition(knservingv1.ServiceConditionReady)
 
-	switch {
-	case serviceCondition == nil:
-	case serviceCondition.Status == v1.ConditionUnknown:
-		conditionSet.Manage(ss).MarkUnknown(conditionType, serviceCondition.Reason, serviceCondition.Message)
-		statusSpec.Hostname = ""
-	case serviceCondition.Status == v1.ConditionTrue:
-		conditionSet.Manage(ss).MarkTrue(conditionType)
-		if serviceStatus.URL != nil {
-			statusSpec.Hostname = serviceStatus.URL.Host
-		}
-	case serviceCondition.Status == v1.ConditionFalse:
-		conditionSet.Manage(ss).MarkFalse(conditionType, serviceCondition.Reason, serviceCondition.Message)
-		statusSpec.Hostname = ""
-	}
+	// switch {
+	// case serviceCondition == nil:
+	// case serviceCondition.Status == v1.ConditionUnknown:
+	// 	conditionSet.Manage(ss).MarkUnknown(conditionType, "serviceCondition.Reason", "string")
+	// case serviceCondition.Status == v1.ConditionTrue:
+	// 	conditionSet.Manage(ss).MarkTrue(conditionType)
+	// case serviceCondition.Status == v1.ConditionFalse:
+	// 	conditionSet.Manage(ss).MarkFalse(conditionType, serviceCondition.Reason, serviceCondition.Message)
+	// }
```

Comment:
This part is commented out. conditionSet.Manage(ss).MarkTrue() will trigger internal error. It remaines to be fixed